docker/nginx/addons.conf

```diff
@@ -11,29 +11,15 @@ server {
     }
 
     location /static/ {
-        alias /srv/static/;
-    }
+         alias /srv/site-static/;
 
-    location /site-static/ {
-        alias /srv/site-static/;
+         error_page 404 = @olympia;
```

Comment:
Add small comment explaining that this is the fallback for files not directly available